main/glossary/README.md

```diff
@@ -109,27 +107,26 @@ The installation operation returns
 an `installation`, which is an object with one method; `getBundle()`. You can access an installed contract's source
 code via `const { source } = await E(installation).getBundle();`.
 In many cases, the bundled source is a single reviewable string.
-In others, the bundle contains to base 64 encoded zip file that you can
+In others, the bundle contains a base64-encoded zip file that you can
```

Comment:
It might be worthwhile to shift the emphasis here. Base64-encoded Zip files are the norm now, not the exception.